terraform/vault.tf

```diff
@@ -0,0 +1,109 @@
+resource "aws_kms_key" "vault" {
+  description             = "Vault unseal key"
+  deletion_window_in_days = 7
+
+  tags = {
+    Name = "${terraform.workspace}-vault"
+  }
+}
+
+resource "aws_instance" "vault" {
```

Comment:
Do we need to create all those resources if `safety_rules_use_vault` set to false?

Reply:
No, but I don't expect to have `safety_rules_use_vault` default to false for very long.